base/applications/network/dwnl/lang/ro-RO.rc

```diff
@@ -0,0 +1,32 @@
+/*
+ * PROJECT:     ReactOS File Download Utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Romanian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Uz: dwnl URL [DESTINAȚIE]\n"
+    IDS_RESOLVING "Rezolvare %s... "
+    IDS_DONE "terminat.\n"
+    IDS_LENGTH_FULL "Lungime: %I64u [%s]\n"
+    IDS_LENGTH "Lungime: %I64u\n"
+    IDS_FILE_SAVED "\nFișier salvat.\n"
```

Comment:
I suggest to avoid using the literal translation of the term "save". In English it has a broader coverage, like "save money" whereas in Romanian it doesn't. Here, considering the context, the best translation I see would be "Fișier descărcat."